ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -2012,5 +2012,14 @@
     "FieldType": "Text",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "otherOrderType",
+    "Label": " ",
+    "FieldType": "Text",
```

Comment:
probably would be better to use YesOrNo type to give a clue what kind of values are possible